### PR TITLE
@ptatian Added the following tables to Acs_sf_2010_14_dc.sas

### PR DESCRIPTION
**Pull request by sdiby**
B03001 Hispanic or Latino by Specific Origin
B05003I – Sex by Age by Nativity + Citizenship Status
B16006 - LANGUAGE SPOKEN AT HOME BY ABILITY TO SPEAK ENGLISH FOR THE
POPULATION 5 YEARS AND OVER (HISPANIC OR LATINO

### CODE REVIEW

**Comment by ptatian** [comment]
@sdiby Reviewing now.

File: Prog/SF_2010_14/Acs_sf_2010_14_dc.sas [hunk 1]
```diff
@@ -34,11 +34,11 @@
   table_list = 
   	B00001 B00002 B01002 B01003 
 	B01001 B01001B B01001C B01001D B01001E B01001F B01001G B01001H B01001I
-	B02001 B03002 B05002 B06002 B06007 B06009 B07012 B09001
+	B02001 B03001 B03002 B05002 B05003I B06002 B06007 B06009 B07012 B09001
     B06004 B06004B B06004C B06004D B06004E B06004F B06004G B06004H B06004I
```
Comment (ptatian) [inline Prog/SF_2010_14/Acs_sf_2010_14_dc.sas]: There is no B06004 table. B06004 should be deleted from this list. (B06004B B06004C etc. are fine.)
The revisions= parameter value should be changed to %str(Add tables for State of Latinos project.)

Comment (sdiby) [inline Prog/SF_2010_14/Acs_sf_2010_14_dc.sas]: I've deleted the B06004 table and updated the revisions= parameter. I also made a note of the update in the header.

Comment (ptatian) [inline Prog/SF_2010_14/Acs_sf_2010_14_dc.sas]: Did you sync your commit?

Comment (sdiby) [inline Prog/SF_2010_14/Acs_sf_2010_14_dc.sas]: I did--but for some reason they're syncing to an old branch that I thought I deleted and renamed to develop-#26. The original branch name is update_acs_table_list. Are you able to merge from that one?

**Pull request merged**